azure-pipelines.yml

```diff
@@ -52,6 +52,8 @@ variables:
 stages:
 - stage: BuildVS
   pool: sonicbld-1es
+  variables:
```

Comment:
azure-pipelines.yml don't need this Variable Group
official-build.yml need this.